generators/entity/templates/src/test/java/package/web/rest/_EntityResourceIntTest.java

```diff
@@ -308,7 +314,8 @@ public void initTest() {<% if (databaseType == 'mongodb' || databaseType == 'cas
     @Transactional<% } %>
     public void update<%= entityClass %>() throws Exception {
         // Initialize the database
-        <%= entityInstance %>Repository.save<% if (databaseType == 'sql') { %>AndFlush<% } %>(<%= entityInstance %>);
+        <%= entityInstance %>Repository.save<% if (databaseType == 'sql') { %>AndFlush<% } %>(<%= entityInstance %>);<% if (searchEngine == 'elasticsearch') { %>
+        <%= entityInstance %>SearchRepository.save(<%= entityInstance %>);<%_ } _%>
```

Comment:
You are right. I just wanted to keep the code similar to the other tests.. But I can change that if you prefer service invocation?